src/Service/DatabaseService.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+
```

Comment:
Declare in the all files

Reply:
This branch was created before I you create this task, then in master already added declare.

Reply:
But it already is in master branch

Reply:
But it already is in master

Reply:
Git rejects pushing new commits, ask merge at first.